pkg/cli/bicep/build.go

```diff
@@ -94,22 +95,6 @@ func runBicepJson(args ...string) (map[string]any, error) {
 	return template, err
 }
 
-// Build the provided `.bicep` file and returns the deployment template.
-//
-
-// Build() reads a Bicep file at the given file path and returns a map of the compiled output and an error if the
-// compilation fails.
-func Build(filePath string) (map[string]any, error) {
-	// rad-bicep is being told to output the template to stdout and we will capture it
-	// rad-bicep will output compilation errors to stderr which will go to the user's console
-	return runBicepJson("build", "--stdout", filePath)
```

Comment:
I had to change the code here because it was always using the `--stdout` option. For the flux controller, I needed to run without this option.